### PR TITLE
Añadida la conexion entre despensa y lista de la compra

### PR DESCRIPTION
**Pull request by uo289549**

### CODE REVIEW

**Review by didierrc — APPROVED** [review]
Everything is OK!

File: app/src/main/java/com/example/compicomida/model/GroceryRepository.kt [hunk 1]
```diff
@@ -77,5 +77,18 @@ class GroceryRepository(
         return db.groceryListDao.getById(listID)
     }
 
+    suspend fun deletePurcharedItems(listId: Int){
```
Comment (Raulms29) [inline app/src/main/java/com/example/compicomida/model/GroceryRepository.kt:80]: Change the name to purchased, not purchared.

File: app/src/main/java/com/example/compicomida/views/activities/grocery/GroceryItemsListActivity.kt [hunk 2]
```diff
@@ -93,6 +94,18 @@ class GroceryItemsListActivity : AppCompatActivity() {
         listGroceryItemsViewModel.refreshGroceryItems()
     }
 
+//    private fun confirmOnClickListener(){
```
Comment (Raulms29) [inline app/src/main/java/com/example/compicomida/views/activities/grocery/GroceryItemsListActivity.kt:97]: Remove commented code if it's not useful.

**Review by Raulms29 — CHANGES_REQUESTED** [review]
I found some minor things. A typo and some commented code.

**Review by Raulms29 — APPROVED** [review]

**Pull request merged**